src/components/style.css

```diff
@@ -1,4 +1,292 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto+Slab:wght@300;500;700;800&display=swap');
+
 * {
   margin: 0;
   padding: 0;
 }
+
+:root {
+  --bubble-gum-pink: #ff5ab7;
+  --black: #1e1e1e;
+  --pinkish-grey: #c7c7c7;
+  --very-light-pink: #c4c4c4;
+  --pale-grey: #f5f6fa;
+  --z: #fafafa;
+  --medium-blue: #307bbe;
+  --cool-blue: #4386bf;
+  --white-two: #fff;
+  --azure: #0290ff;
+  --e8: #e8e8e8;
+}
+
+.App {
+  width: 100%;
+  height: 68.563rem;
+  margin: 0 2.5rem 4.063rem 0;
+  padding: 0 0 49.625rem;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  background-color: var(--z);
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  background-color: var(--white-two);
+}
+
+.section1 {
+  display: flex;
+}
+
+.heading {
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  padding-right: 2.938rem;
+  font-family: Montserrat, Arial, Helvetica, sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--azure);
+}
+
+.navbar ul {
+  padding-top: 2rem;
+}
+
+.links {
+  padding: 1.438rem;
+  text-decoration: none;
+  margin-right: 2rem;
+  opacity: 0.5;
+  font-family: Montserrat, Arial, Helvetica, sans-serif;
+  font-size: 0.813rem;
```

Comment:
- [ ] This line is repeated in multiple selectors. kindly consider making it a class and then give that class to any element you want as I suggested above. This way we can follow the best practices and DRY principle.

Reply:
Font family added to body now. Thanks.